src/applications/check-in/components/PrepareContent.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { makeSelectFeatureToggles } from '../utils/selectors/feature-toggles';
+import { useFormRouting } from '../hooks/useFormRouting';
+
+const PrepareContent = props => {
+  const { smallHeading, router, appointmentCount } = props;
+  const { jumpToPage } = useFormRouting(router);
+  const { t } = useTranslation();
+  const selectFeatureToggles = useMemo(makeSelectFeatureToggles, []);
+  const { isMedicationReviewContentEnabled } = useSelector(
+    selectFeatureToggles,
+  );
+
+  const onPrepareClick = useCallback(
+    e => {
+      e.preventDefault();
+      jumpToPage('what-to-bring-to-va-health-care-appointments');
+    },
+    [jumpToPage],
+  );
+  if (isMedicationReviewContentEnabled) {
+    return (
+      <div data-testid="prepare-content">
+        <h2
+          className={
+            smallHeading ? 'vads-u-font-size--sm' : 'vads-u-margin-top--0'
+          }
+        >
+          {t('prepare-for-your-appointment', { count: appointmentCount })}
+        </h2>
+        <p>{t('bring-insurance-cards-list-medications-other')}</p>
+        <p className="vads-u-margin-bottom--4">
```

Comment:
The margin bottom here and on the other links are causing some extra spacing on the complete view. Might be able to remove it but It will likely cause issues in other components where it's the only link.

Reply:
nice catch! I was just matching the existing link spacing, I didn't notice that it got tighter